### ISSUE TITLE
Identification error

### ISSUE THREAD

**Issue by SnaiCrys** [ImagicTheCat/vRP#321]
```
Connecting: Le
[profile] => playerConnecting
[vRP] MySQL begin query vRP/userid_byidentifier (76094)
[profile] <= playerConnecting 0.0020540000000438s
[vRP] LeaveMeAlone (**.185) rejected: identification error
[profile] => vRP save datatables
[profile] <= vRP save datatables 4.5000000227446e-05s
[profile] => playerDropped
[profile] <= playerDropped 0.00012400000014168s
```
Nobody can connect to the server. Server works 15 hours.

**Comment by ImagicTheCat** [comment]
As you can see, there is no "end query". So vrp_mysql is faulted. Most of the vRP issues right now are about vrp_mysql, and I can't figure what is causing exceptions/errors in the C# code.

**Comment by SnaiCrys** [comment]
Yes, I saw. If you want to test vrp_mysql on a full server with debug logs, I will help.

**Comment by BobbyLinny** [comment]
Try enabling these two settings below, they seemed to have helped my server with the error, or maybe it's just calmed down for some reason?

sv_authMaxVariance 1
sv_authMinTrust 5

**Comment by SnaiCrys** [comment]
fixed after 370fcc9e459eaf881be57fc23813f563c2519f37
